### PR TITLE
Add sdk-svelte

### PR DESCRIPTION
**Pull request by DTCurrie**
Just getting thing setup 👷 🏗️

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      - name: Build 🏗️
+        run: pnpm run build
+
+      - name: Lint and type check 💅
+        run: pnpm run check
+
+      - name: Upload npm @viamrobotics/sdk-svelte artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: npm-core-dist
```
Comment (ethanlook) [inline .github/workflows/ci.yml:29]: nit: `npm-sdk-svelte-dist` here and L65

**Review by ethanlook — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (DTCurrie) [reply .github/workflows/ci.yml:29]: Ok cool I wasn't sure if that was a `npm` specific thing or not.

Comment (ethanlook) [reply .github/workflows/ci.yml]: Nope - our name for the artifact

**Review by micheal-parks — APPROVED** [review]
🚀 🚀 🚀

**Comment by DTCurrie** [comment]
Should we be expecting this run to fail?

https://github.com/viamrobotics/bmo/actions/runs/6657495297/job/18092384009?pr=1

It seems like it is running the `install` but then cannot find `node_modules`?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install 🔧
+        uses: viamrobotics/js-config/.github/actions/setup@0006127efb3f62669f662ca31c555a49a034935e
```
Comment (mcous) [inline .github/workflows/ci.yml:18]: Since we control this action, I think `@main` is safe

File: packages/sdk-svelte/.gitignore
Comment (mcous) [inline packages/sdk-svelte/.gitignore:1]: Is this file necessary?

File: packages/sdk-svelte/svelte.config.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    typescript: {
+      config: (tsconfig) => ({
+        ...tsconfig,
+        include: [
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+          ...tsconfig.include,
+          '../.eslintrc.cjs',
+          '../.prettierrc.cjs',
+          '../svelte.config.js',
+          '../playwright.config.ts',
```
Comment (mcous) [inline packages/sdk-svelte/svelte.config.js:24]: Why are these paths all `../` instead of `./`?

File: packages/sdk-svelte/playwright.config.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+
+const config: PlaywrightTestConfig = {
+  webServer: {
+    command: 'npm run build && npm run preview',
+    port: 4173,
+  },
```
Comment (mcous) [inline packages/sdk-svelte/playwright.config.ts:7]: Should this use `pnpm` rather than `npm`?

**Review by mcous — COMMENTED** [review]
A few little things to grab in future PRs

File: packages/sdk-svelte/svelte.config.js (hunk 3 above)
Comment (DTCurrie) [reply packages/sdk-svelte/svelte.config.js:24]: I am actually not sure, I pulled a lot of this from `prime-core`: https://github.com/viamrobotics/prime/blob/main/packages/core/svelte.config.js

I'll remove them and see what happens.

File: packages/sdk-svelte/.gitignore
Comment (DTCurrie) [reply packages/sdk-svelte/.gitignore:1]: We just need to add the `.svelte-kit` ignore, otherwise the rest is redundant.
